hybrid/compartment_examples/inter_comp_call/README.md

```diff
@@ -0,0 +1,18 @@
+# "inter_comp_call" examples
+
+These examples are not automatically built by the top-level makefiles, but are
+built in a similar manner.
+
+* to run the `base` example:
+
+```
+$ cd base && make -f Makefile.morello-hybrid run-main
+```
+
+* to run one of the `secure` examples:
+
+```
+$ cd secure && make -f Makefile.morello-hybrid run-inter_comp_call-<example>
+```
+
+Refer to the top-level [README](../../../../README.md) for usage details.
```

Comment:
I think there's one too many `..`.

Reply:
Good catch. Fixed in https://github.com/capablevms/cheri-examples/pull/76/commits/faa109b36b3f4a4245e8cb88c73585b2b10bbabe